src/components/pages/music.tsx

```diff
@@ -49,26 +49,29 @@ export interface FormData {
 }
 
 const MusicForm: React.FC<MusicFormProps> = ({ onSubmit }) => {
+  const { translations } = useLanguage();
   const navigate = useNavigate();
   const [id, setId] = useState<string>(new Date().getTime().toString());
   const {
     control,
     register,
     handleSubmit,
     reset,
+    trigger,
     formState: { errors },
   } = useForm<FormData>({ defaultValues: { id } });
 
   const { setFormData } = useFormData();
 
-  const onSubmitHandler = (data: FormData) => {
+  const onSubmitHandler: SubmitHandler<FormData> = (data) => {
     if (onSubmit) {
       onSubmit(data);
       console.log("Form data submitted:", data);
     }
     setFormData((prevData) => [...prevData, data]);
-    navigate("/cards");
+    navigate("/playlist");
```

Comment:
Move all routes in separate file, so you can export them from it
So do everywhere you use hardcoded route